### ISSUE TITLE
No such file `.gitignore` exception

### ISSUE THREAD

**Issue by dlukeomalley** [returntocorp/bento#102]
**Describe the bug**
When running `bento init`, I get an exception because no `.gitignore` file exists in the project.

**Expected result**
Bento initializes my project.

**Actual result**
```bash
$bento init
Creating default configuration at .bento.yml
✔ Created .bento.yml. Please check this file in to source control.

Detected project with Python

✔ Creating virtual environment for r2c.bandit
✔ Python package bandit will be installed
✔ Creating virtual environment for r2c.flake8
✔ Python package flake8 will be installed
✔ Python package flake8-json will be installed
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/bento/__main__.py", line 14, in main
    cli()
  File "/Users/luke/Library/Python/3.7/lib/python/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/luke/Library/Python/3.7/lib/python/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/luke/Library/Python/3.7/lib/python/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/luke/Library/Python/3.7/lib/python/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/luke/Library/Python/3.7/lib/python/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/bento/cli.py", line 535, in init
    with open(ignore_file, "r") as fd:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/luke/git/tmp/project-python-django-webapp/.gitignore'
✘ There was an exception [Errno 2] No such file or directory: '/Users/luke/git/tmp/project-python-django-webapp/.gitignore'
```

**Additional context**
Version: bento/0.1.1b3
Project: project-python-django-webapp

**Comment by tnarla** [comment]
question: is there a gitignore in a parent directory?

**Comment by tnarla** [comment]
https://github.com/returntocorp/bento-core/pull/87

**Comment by nbrahms** [comment]
Hello... this is fixed in 0.1.1b4
